code/modules/plumbing/ducts.dm

```diff
@@ -166,8 +166,9 @@ All the important duct code:
 	lose_neighbours()
 	reset_connects(0)
 	update_appearance()
-	if(ispath(drop_on_wrench) && !QDELING(src))
+	if(ispath(drop_on_wrench))
 		new drop_on_wrench(drop_location())
+	if(!QDELING(src))
 		qdel(src)
```

Comment:
Isn't this equivalent to `qdel(src)` unconditionally?

Reply:
no not at all. It could/should be QDELETING(), but that's totally unrelated